packages/cosmic-swingset/src/chain-main.js

```diff
@@ -502,14 +505,30 @@ export default async function main(progname, args, { env, homedir, agcc }) {
       portNums.lien = action.lienPort;
     }
 
+    // Ensure that initialization has completed.
     if (!blockingSend) {
       const { savedChainSends: scs, ...fns } =
         await launchAndInitializeSwingSet(action);
+
+      const toStorage = message => {
+        return fns.bridgeOutbound(BRIDGE_ID.STORAGE, message);
+      };
+      const rootStorageNode = makeChainStorageRoot(
+        toStorage,
+        'swingset',
+        'published',
+      );
+      const installationStorageNode = rootStorageNode.makeChildNode(
+        'installation',
+        { sequence: true },
+      );
```

Comment:
```suggestion
      const rootStorageNode = makeChainStorageRoot(
        toStorage,
        'swingset',
        STORAGE_PATH.CUSTOM,
      );
      const installationStorageNode = rootStorageNode.makeChildNode(
        'installation',
        { sequence: true },
      );
```

But I wonder if "installation" should be top-level rather than a child of "published", in which case it would get its own constant in packages/vats/src/chain-storage-paths.js.

Reply:
What’s the intended scope of the `published` namespace as opposed to the root of the storage tree? I’m open to abbreviating this to `installed`, `bundles`, or some such.

Reply:
"published" is the catchall namespace that was necessary to prevent collisions with system-relevant paths at "activityhash"/"mailbox"/etc.—it is intended that only a single vat gets access to the "published" storage node, that it can and does create and distribute arbitrarily-named child nodes, and that it does not share the "published" node without a coordination mechanism (basically, that it has complete and singular authority over the "published" subtree, which it partitions and delegates to code that ultimately includes third-party contracts).

Having this independent "published.installation" storage node would impose a requirement that "installation" is itself reserved in that vat, and I'd prefer to avoid having such reservations at multiple levels of the chain storage tree.

Reply:
Perfect. That very clearly indicates that this topic should be top-level.

Reply:
I changed the topic name to merely `bundles` at the top-level.